docs/naps/0-nap-process.md

```diff
@@ -0,0 +1,298 @@
+prototype implementation be co-developed with the NAP, as ideas that sound
+good in principle sometimes turn out to be impractical. Often it makes
+sense for the prototype implementation to be made available as PR to the
+napari repo, as long as it is properly marked as WIP (work in progress).
+
+### Review and Resolution
+
+NAPs are discussed in Zulip, on image.sc, and on GitHub. The possible paths
+of the status of NAPs are as follows:
+
+```{image} _static/nap-flowchart.png
+```
+
+All NAPs should be created with the `Draft` status.
+
+The author of the NAP should periodically update the NAP with new points
+both against and in favor of the NAP raised in discussion.
+
+Eventually, after discussion, there may be a consensus that the NAP
+should be accepted – see the next section for details. At this point
+the status becomes `Accepted`.
+
+Once a NAP has been `Accepted`, the reference implementation must be
+completed. When the reference implementation is complete and incorporated
+into the main source code repository, the status will be changed to
+`Final`.
+
+To allow gathering of additional design and interface feedback before
+committing to long term stability for a language feature or standard
+library API, a NAP may also be marked as "Provisional". This is short for
+"Provisionally Accepted", and indicates that the proposal has been accepted
+for inclusion in the reference implementation, but additional user feedback
+is needed before the full design can be considered "Final". Unlike regular
+accepted NAPs, provisionally accepted NAPs may still be Rejected or
+Withdrawn even after the related changes have been included in a release.
+
+Wherever possible, it is considered preferable to reduce the scope of a
+proposal to avoid the need to rely on the "Provisional" status (e.g. by
+deferring some features to later NAPs), as this status can lead to version
+compatibility challenges in the wider ecosystem.
+
+A NAP can also be assigned status `Deferred`. The NAP author or a core
+developer can assign the NAP this status when no progress is being made on
+the NAP.
+
+A NAP can also be `Rejected`. Perhaps after all is said and done it
+was not a good idea. It is still important to have a record of this
+fact. The `Withdrawn` status is similar---it means that the NAP author
+themselves has decided that the NAP is actually a bad idea, or has
+accepted that a competing proposal is a better alternative.
+
+When a NAP is `Accepted`, `Rejected`, or `Withdrawn`, the NAP should be
+updated accordingly. In addition to updating the status field, at the very
+least the `Resolution` header should be added with a link to the relevant
+post on the discussion forum.
+
+NAPs can also be `Superseded` by a different NAP, rendering the
+original obsolete. The `Replaced-By` and `Replaces` headers
+should be added to the original and new NAPs respectively.
+
+Process NAPs may also have a status of `Active` if they are never
```

Comment:
Is there an "Inactive" status for process NAPs that are no longer active or have been replaced?

Reply:
Evidently not! Again, it's important to note that this process is not something we came up with but inherited from Python and the rest of the ecosystem. I would like to merge this with minimal changes so that we have it in place, then potentially create new NAPs to improve the process.